API/TreeData/TreeDataMock.cs

```diff
@@ -0,0 +1,149 @@
+using System;

```

Comment:
Rename `API` directory to `WebApi`. Be careful when renaming to use `git mv` to preserve git history.

Reply:
Fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/a936400665e8815bd67d474b006b44b0b214fd84